ct/redis.sh

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env bash
+source <(curl -s https://raw.githubusercontent.com/tteck/Proxmox/main/misc/build.func)
+# Copyright (c) 2021-2024 tteck
+# Author: tteck (tteckster)
+# License: MIT
+# https://github.com/tteck/Proxmox/raw/main/LICENSE
+
+function header_info {
+clear
+cat <<"EOF"
+  ____           ___     
```

Comment:
fix

Reply:
This is taken from here .. https://www.patorjk.com/software/taag/#p=display&f=Slant&t=Redis .. Not clear what should I fix here.